emerge/output/html/resources/js/emerge_git.js

```diff
@@ -110,7 +110,7 @@ function calculateAuthorsForDateRange() {
 }
 
 function addGitMetricToFileNodes() {
-    if (currentGraphType.includes('file_result_dependency_graph')) {
+    if (currentGraph && currentGraphType.includes('file_result_dependency_graph')) {
```

Comment:
For languages where entity parsing is supported, this evaluated to false since `currentGraphType` defaults to `"entity_result_dependency_graph"` when called from `initAppUI`, but for languages where entity parsing is _NOT_ supported, it defaults to `"file_result_dependency_graph"` and fails since `currentGraph` is not fully initialized.